homeassistant/components/samsungtv/media_player.py

```diff
@@ -204,13 +204,17 @@ async def async_update(self) -> None:
             )
 
         if self._attr_state != STATE_ON:
+            if self._dmr_device:
```

Comment:
Should this check if it’s subscribed first?

Reply:
We could do it like this:
```suggestion
            if self._dmr_device and self._dmr_device.is_subscribed:
```